Geometry_Engine/Modify/SplitAtPoints.cs

```diff
@@ -212,111 +212,113 @@ public static List<PolyCurve> SplitAtPoints(this PolyCurve curve, List<Point> po
                 return new List<PolyCurve> { curve.DeepClone() };
```

Comment:
This `DeepClone` makes me upset, especially that there is a chance it was me who put it here long time ago 🙈

Reply:
Yeah, prefer not to change it now though, as could impact elsewhere...